### PR TITLE
Several small fixes

### PR DESCRIPTION
**Pull request by torbenal**
- Remove outliers
- More readable variable names shown on dashboard
- Re-introduce previously filtered columns, i.e. years experience and tools
- Add link to Github repo

### CODE REVIEW

File: .github/workflows/black.yml [hunk 1]
```diff
@@ -13,4 +13,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: jpetrucciani/black-check@21.12b0
```
Comment (saattrupdan) [inline .github/workflows/black.yml:16]: This change means that we constantly have to update the `black` version in our requirements. Isn't it easier if we just fix a version?

File: requirements.txt [hunk 2]
```diff
@@ -12,4 +12,4 @@ streamlit==1.7.0
 scipy>=1.7.3
 
 # Style
-black==21.12b0
```
Comment (saattrupdan) [inline requirements.txt:15]: Either we do not fix the version, i.e. just write `black` here, or we update the workflow to be version `22.3.0`.

File: src/utils.py [hunk 3]
```diff
@@ -25,6 +48,12 @@
     "danish_national": ["Yes", "No"],
```
Comment (saattrupdan) [inline src/utils.py:48]: This column shouldn't be included at all?

**Review by saattrupdan — CHANGES_REQUESTED** [review]
Godt arbejde @torbenal! Jeg har kun et par småting.

File: .github/workflows/black.yml (hunk 1 above)
Comment (torbenal) [reply .github/workflows/black.yml:16]: Ah, true! I'll change the requirements

**Review by saattrupdan — APPROVED** [review]
LGTM!

**Pull request merged**